### DISCUSSION TITLE
How many members can be added to a GitHub organization?

### DISCUSSION THREAD

**Discussion by kimyeoungrok** [community/community#111254] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello! I'm planning to create a GitHub organization, and it seems like there will be many collaborators. Just out of curiosity, I'd like to ask: What is the maximum number of members that can be added to a GitHub organization?

**Comment by SebastienXia** [comment]
Hi, @kimyeoungrok 

![image](https://github.com/community/community/assets/105004912/28cd7cc9-40f7-4429-a60d-ab7ef7ff3b87)
by Google search ❤️

`[close the discussion if this helped you with your problem]`

**Reply by kimyeoungrok** [reply]
Thank you😁

**Comment by himeshx** [comment]
The umber of members that can be added to a GitHub organization varies based on the GitHub plan the organization has chosen. For GitHub Team plans, organizations can scale their team size with plans ranging in between 10 to 25,000 users, providing more and more robust collaboration features and advanced tools for medium-sized to large teams. 

ps:  Kindly consider marking the provided response as the answer if it has addressed your query satisfactorily.
